.vitest/src/instances.ts

```diff
@@ -2,14 +2,14 @@ import getPort from "get-port";
 import { createServer } from "prool";
 import { anvil, rundler } from "prool/instances";
 import { createClient, http, type Chain, type ClientConfig } from "viem";
-import { arbitrumSepolia } from "viem/chains";
+import { localhost } from "viem/chains";
 import { split } from "../../aa-sdk/core/src/transport/split";
 import { poolId, rundlerBinaryPath } from "./constants";
 import { paymasterTransport } from "./paymaster";
 
-export const anvilArbSepolia = defineInstance({
-  chain: arbitrumSepolia,
-  forkUrl: "https://arbitrum-sepolia-rpc.publicnode.com",
+export const localInstance = defineInstance({
+  chain: localhost,
+  forkUrl: "https://ethereum-sepolia-rpc.publicnode.com",
```

Comment:
just using this to fork some state, but using localhost above because prool doesn't let me set up rundler with the chain id. will need to fix that

Reply:
We can avoid forking by making sure we deploy all the contracts we need, so we can defo migrate off of it, but I wanted to avoid all of that complexity for now and just use this.

In the future, what we'll do is deploy the state that we need. Viem takes the same approach we do here which is to use a forkUrl for chains. Ideally the fork url matches the chain we pass in too, so here we would ideally be passing in sepolia.

One nice thing about using a forkUrl is in the future we can pin the fork to a specific block, I'll tighten things up and add more features once we have 1:1 testing parity